ui/app/AppLayouts/Profile/controls/ShowcaseDelegate.qml

```diff
@@ -62,67 +62,94 @@ StatusDraggableListItem {
 
             Layout.maximumWidth: root.width *.4
         },
-        DisabledTooltipButton {
+        StatusButton {
             interactive: root.contextMenuEnabled
-            tooltipText: root.tooltipTextWhenContextMenuDisabled
-            buttonComponent: StatusRoundButton {
-                enabled: root.contextMenuEnabled
-                icon.name: ProfileUtils.visibilityIcon(root.showcaseVisibility)
-                width: 58
-                height: 30
-                border.width: 1
-                border.color: Theme.palette.directColor7
-                radius: 14
-                highlighted: menuLoader.item && menuLoader.item.opened
-                onClicked: {
-                    menuLoader.active = true
-                    menuLoader.item.popup(width - menuLoader.item.width, height)
+            tooltip.text: root.tooltipTextWhenContextMenuDisabled
+            icon.name: ProfileUtils.visibilityIcon(root.showcaseVisibility)
+            horizontalPadding: Style.current.halfPadding
+            verticalPadding: 3
+            Layout.preferredWidth: 72
+            Layout.preferredHeight: root.height/2
+            radius: height/2
+            highlighted: menuLoader.item && menuLoader.item.opened
+            onClicked: {
+                menuLoader.active = true
+                menuLoader.item.popup(width - menuLoader.item.width, height)
+            }
+            text: "    " // NB to give the icon and indicator some even space
+
+            indicator: StatusIcon {
+                anchors.right: parent.right
+                anchors.rightMargin: parent.horizontalPadding
+                anchors.verticalCenter: parent.verticalCenter
+                icon: "chevron-down"
+                color: parent.interactive ? Theme.palette.primaryColor1 : Theme.palette.baseColor1
+            }
+
+            ButtonGroup {
+                id: showcaseVisibilityGroup
+                exclusive: true
+                onClicked: function(button) {
+                    const newVisibility = (button as ShowcaseVisibilityAction).showcaseVisibility
+                    if (newVisibility !== root.showcaseVisibility)
+                        root.showcaseVisibilityRequested(newVisibility)
                 }
+            }
 
-                ButtonGroup {
-                    id: showcaseVisibilityGroup
-                    exclusive: true
-                    onClicked: function(button) {
-                        const newVisibility = (button as ShowcaseVisibilityAction).showcaseVisibility
-                        if (newVisibility !== root.showcaseVisibility)
-                            root.showcaseVisibilityRequested(newVisibility)
+            Loader {
+                id: menuLoader
+                active: false
+                sourceComponent: StatusMenu {
+                    onClosed: menuLoader.active = false
+                    StatusMenuHeadline { text: qsTr("Show to") }
+
+                    ShowcaseVisibilityAction {
```

Comment:
I guess here you are duplicating the actions :). Something needs to be removed:

 ![Screenshot 2024-03-14 at 14 44 10](https://github.com/status-im/status-desktop/assets/97019400/eec369b4-9324-49bb-a166-0cae04fd68c2)

Reply:
Fixed, was a bad rebase :/

Reply:
Perfect!! Tested again and it looks fine!!